packages/eddsa-poseidon/rollup.config.ts

```diff
@@ -14,19 +16,53 @@ const banner = `/**
  * @see [Github]{@link ${pkg.homepage}}
 */`
 
-export default {
-    input: "src/index.ts",
-    output: [
-        { file: pkg.exports["."].require, format: "cjs", banner },
-        { file: pkg.exports["."].default, format: "es", banner }
-    ],
-    external: Object.keys(pkg.dependencies),
-    plugins: [
-        typescript({ tsconfig: "./build.tsconfig.json" }),
-        commonjs(),
-        nodeResolve({
-            preferBuiltins: true
-        }),
-        cleanup({ comments: "jsdoc" })
-    ]
-}
+const name = pkg.name.split("/")[1].replace(/[-/]./g, (x: string) => x.toUpperCase()[1])
+
+export default [
+    {
+        input: "src/index.ts",
+        output: [
+            { file: pkg.exports["."].require, format: "cjs", banner },
+            { file: pkg.exports["."].default, format: "es", banner }
+        ],
+        external: [
```

Comment:
Why are we putting them as external?

Reply:
Because we don't want them to be included in the final bundle. They should be external dependencies.